serverless/cmd/sequence/main.go

```diff
@@ -39,13 +39,27 @@ var (
 
 func main() {
 	flag.Parse()
+	var err error
+
+	toAdd, err := filepath.Glob(*entries)
+	if err != nil {
+		glog.Exitf("Failed to glob entries %q: %q", *entries, err)
+	}
+	if toAdd == nil {
+		glog.Exitf("Sequence must be run with at least one valid entry")
```

Comment:
Can use `glog.Exit` here (no `f`) since this is just static text (no format specifiers/params)